src/components/Multiselect/EllipsisedOption.vue

```diff
@@ -73,7 +71,7 @@ export default {
 		/**
 		 * Index at which to split the name if it is longer than 10 characters.
 		 *
-		 * @returns {Integer} The position at which to split
+		 * @return {bigint} The position at which to split
```

Comment:
`number` ?

Reply:
Here it really is an integer. Shall we still make it a number?

Reply:
Hmm, I don't know, `integer` is not a valid `js` type. Not sure if we should use it.

Reply:
Yes, `integer` is not, but `bigint` is. Anyway, I changed it to number everywhere.

Reply:
You are right, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt.

But the returned value will probably not be a `BigInt` so it would be misleading.